manifests/julia/traefik-services.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-external-np
+  namespace: traefik
+  labels:
+    app: traefik
+spec:
+  type: NodePort
+  selector:
+    app: traefik
+  ports:
+    - name: http
+      port: 80
+      nodePort: 30080
+    - name: https
+      port: 443
+      nodePort: 30443
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-external
+  namespace: traefik
+  labels:
+    app: traefik
+spec:
+  type: LoadBalancer
+  loadBalancerIP: 192.168.86.200
```

Comment:
my `traefik-services.yaml` file (camh) has two extra load balancers. the internal one (.201) probably wont work for you without your own internal DNS, but the traefik dashboard one (.202) should I think

Reply:
same here.